code/modules/events/portal_storm.dm

```diff
@@ -1,7 +1,7 @@
 /datum/round_event_control/portal_storm_syndicate
 	name = "Portal Storm: Syndicate Shocktroops"
 	typepath = /datum/round_event/portal_storm/syndicate_shocktroop
-	weight = 2
+	weight = 1
```

Comment:
I'd keep the same or even up a little bit but up the min players to 45 or something